--- upstream/specs/libthruk.spec	2026-05-11 23:53:35.000000000 +0200
+++ specs/libthruk.spec	2026-05-12 10:30:21.033674417 +0200
@@ -1,14 +1,18 @@
-Summary: Thruk perl libraries
-Name: libthruk
-Version: 3.26
-Release: 23.17
-License: GPL-2.0-or-later
-Group: Applications/System
-URL: http://www.thruk.org/
-Packager: Sven Nierlein <sven.nierlein@consol.de>
-Vendor: Labs Consol
-Source0: %{name}-%{version}.tar.gz
-BuildRoot: %{_tmppath}/%{name}-%{version}-%{release}
+Summary:       Thruk perl libraries
+Name:          libthruk
+Version:       3.24
+Release:       1%{?dist}
+License:       GPL-2.0-or-later
+Group:         42/addon/naemon
+
+URL:           http://www.thruk.org/
+Source0:       https://github.com/sni/thruk_libs/archive/refs/tags/v%{version}/%{name}-%{version}.tar.gz
+# this needs to be updated for every version change
+%global src0sum 72d6cf46d1b691ddf2d234cfcb538cb3
+
+BuildRoot:     %{_tmppath}/%{name}-%{version}-%{release}
+
+BuildRequires: xxhash
 BuildRequires: make
 BuildRequires: rsync
 BuildRequires: gcc
@@ -35,6 +39,7 @@
 BuildRequires: epel-release
 %endif
 
+Requires:      yumrepo-epel-thruk
 Requires:      perl(Bit::Vector)
 Requires:      perl(Cpanel::JSON::XS)
 Requires:      perl(Crypt::Rijndael)
@@ -87,7 +92,8 @@
 large installations.
 
 %prep
-%setup -q
+echo "%{src0sum}  %{SOURCE0}" | xxh128sum -c
+%setup -q -n thruk_libs-%{version}
 
 %build
 %{__make}
@@ -104,7 +110,36 @@
 
 %changelog
 * Fri Aug 22 2025 Sven Nierlein <sven.nierlein@consol.de> 3.24-1
+- now based on upstream spec 3.24
+- group changed to 42/addon/naemon
+- use git source url and save xxh128 hash in the spec file and check it
+- yumrepo-epel-thruk for perl dependencies
+
+* Fri Aug 22 2025 Sven Nierlein <sven.nierlein@consol.de> 3.24-1
 - Migrate to use system perl modules whenever possible
 
+* Tue Apr 28 2026 Peter Tuschy <foss+rpm@bofh42.de> - 3.22.2-1
+- upstream update 3.22.2
+
+* Tue Mar 24 2026 Peter Tuschy <foss+rpm@bofh42.de> - 3.20-2
+- changed group to 42/addon/naemon for my new repo scripts
+
+* Mon Mar 03 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.20-1
+- upstream update
+
+* Fri Dec 06 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.14-1
+- added el9 Requires perl(Time::HiRes)
+
+* Sat Nov 30 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.14-1
+- reduced extra Requires for el9 to perl(threads), perl(File::Copy)
+
+* Tue Nov 26 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.14-1
+- reset release number to 1 for my own repo
+- use git source url and save xxh128 hash in the spec file and check it
+- added a bunch of perl modules for build and install on el9
+
+* Thu Oct 24 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.14-19
+- added optional macro dist to release
+
 * Mon Jul 13 2015 Sven Nierlein <sven.nierlein@consol.de> 2.00-1
 - Initial libs package
