diff -Nau upstream/specs/naemon-livestatus.spec specs/naemon-livestatus.spec
--- upstream/specs/naemon-livestatus.spec	2026-03-24 09:48:49.822325610 +0100
+++ specs/naemon-livestatus.spec	2026-04-28 21:56:29.414718837 +0200
@@ -1,14 +1,20 @@
-Summary: Naemon Livestatus Eventbroker Module
-Name: naemon-livestatus
-Version: 1.4.3
-Release: 16.13
-License: GPL-2.0-only
-Group: Applications/System
-URL: https://www.naemon.io/
-Packager: Naemon Core Development Team <naemon-dev@monitoring-lists.org>
-Vendor: Naemon Core Development Team
-Source0: http://labs.consol.de/naemon/download/naemon-livestatus-%{version}.tar.gz
-BuildRoot: %{_tmppath}/naemon-%{version}-%{release}
+%global debug_package %{nil}
+
+Summary:       Naemon Livestatus Eventbroker Module
+Name:          naemon-livestatus
+Version:       1.4.4
+Release:       1%{?dist}
+License:       GPL-2.0-only
+Group:         42/addon/naemon
+
+URL:           https://www.naemon.io/
+Source0:       https://github.com/naemon/%{name}/archive/refs/tags/v%{version}/%{name}-%{version}.tar.gz
+# this needs to be updated for every version change
+%global src0sum 29e0e7a4e0f15c86fe3958e2e636007f
+
+BuildRoot:     %{_tmppath}/%{name}-%{version}-%{release}
+
+BuildRequires: xxhash
 BuildRequires: naemon-devel
 BuildRequires: autoconf
 BuildRequires: automake
@@ -26,6 +32,8 @@
 
 
 %prep
+echo "%{src0sum}  %{SOURCE0}" | xxh128sum -c
+
 %setup -q
 
 
@@ -114,5 +122,21 @@
 %attr(0644,naemon,naemon) %config(noreplace) %{_sysconfdir}/naemon/module-conf.d/livestatus.cfg
 
 %changelog
+* Tue Apr 28 2026 Peter Tuschy <foss+rpm@bofh42.de> - 1.4.4-1
+- upstream update 1.4.4
+
+* Tue Mar 24 2026 Peter Tuschy <foss+rpm@bofh42.de> - 1.4.3-2
+- changed group to 42/addon/naemon for my new repo scripts
+
+* Mon Mar 03 2025 Peter Tuschy <foss+rpm@bofh42.de> - 1.4.3-1
+- upstream update
+
+* Tue Nov 26 2024 Peter Tuschy <foss+rpm@bofh42.de> - 1.4.2-1
+- reset release number to 1 for my own repo
+- use git source url and save xxh128 hash in the spec file and check it
+
+* Thu Oct 24 2024 Peter Tuschy <foss+rpm@bofh42.de> - 1.4.2-15
+- added optional macro dist to release
+
 * Tue Sep 19 2017 Sven Nierlein <sven.nierlein@consol.de> 1.0.7-1
 - Decouple core and livestatus
