diff -Nau upstream/specs/thruk.spec specs/thruk.spec
--- upstream/specs/thruk.spec	2026-03-24 09:48:49.823325619 +0100
+++ specs/thruk.spec	2026-03-24 17:18:07.438574547 +0100
@@ -8,21 +8,27 @@
 %define apachedir httpd
 %endif
 
+Summary:       Monitoring Webinterface for Nagios/Naemon/Icinga and Shinken
 Name:          thruk
 Version:       3.20.2
-Release:       43.13
+Release:       2%{?dist}
 License:       GPL-2.0-or-later
-Packager:      Sven Nierlein <sven.nierlein@consol.de>
-Vendor:        Labs Consol
+Group:         42/addon/naemon
+
 URL:           http://thruk.org
-%define fullname %{name}-%{version}
-Source0:       %{fullname}.tar.gz
+Source0:       https://github.com/sni/Thruk/archive/refs/tags/v%{version}/%{name}-%{version}.tar.gz
+# this needs to be updated for every version change
+%global src0sum 95d047d4ac6b7651cb1439ca05a81d35
+
 BuildRoot:     %{_tmppath}/%{name}-%{version}-%{release}
-Group:         Applications/Monitoring
+
+BuildRequires: xxhash
+BuildRequires: nodejs, npm
 BuildRequires: autoconf, automake, perl, patch
-Summary:       Monitoring Webinterface for Nagios/Naemon/Icinga and Shinken
-AutoReqProv:   no
 BuildRequires: libthruk >= 2.44.2
+
+AutoReqProv:   no
+
 Requires:      thruk-base = %{version}-%{release}
 Requires:      thruk-plugin-reporting = %{version}-%{release}
 %if 0%{?suse_version} < 1315
@@ -47,11 +53,16 @@
 
 %package base
 Summary:     Thruk Gui Base Files
-Group:       Applications/System
+Group:       42/addon/naemon
 Requires:    libthruk >= 2.44.2
 Requires(preun): libthruk
 Requires(post): libthruk
-Requires:    perl logrotate gd wget
+%if 0%{?rhel} >= 8 || 0%{?fedora} >= 28
+Requires:    perl-interpreter
+%else
+Requires:    perl
+%endif
+Requires:    logrotate gd wget
 AutoReqProv: no
 
 #sles and opensuse
@@ -90,7 +101,7 @@
 
 %package plugin-reporting
 Summary:     Thruk Gui Reporting Addon
-Group:       Applications/System
+Group:       42/addon/naemon
 Requires:    %{name}-base = %{version}-%{release}
 %if %{defined suse_version}
 Requires:    xorg-x11-fonts
@@ -104,7 +115,17 @@
 and event reporting.
 
 %prep
-%setup -q -n %{fullname}
+echo "%{src0sum}  %{SOURCE0}" | xxh128sum -c
+%setup -q
+
+# pin npm package versions for themes buid
+sed -i \
+  -e 's|tailwindcss@latest |tailwindcss@3.4.17 |' \
+  -e 's|postcss@latest |postcss@8.4.49 |' \
+  -e 's|autoprefixer@latest |autoprefixer@10.4.20 |' \
+  -e 's|postcss-import@latest |postcss-import@16.1.0|' \
+  -e 's|@tailwindcss/forms@latest |@tailwindcss/forms@0.5.10 |' \
+  themes/themes-available/Light/Makefile
 
 %build
 export PERL5LIB=/usr/lib/thruk/perl5:/usr/lib64/thruk/perl5
@@ -130,6 +151,14 @@
 test -f themes/themes-available/Light/stylesheets/Light.css || %{__make} themes
 test -f themes/themes-available/Light/stylesheets/Light.css || exit 1
 
+# cleanup themes build artefacts
+rm -rf themes/themes-available/*/Makefile \
+       themes/themes-available/*/node* \
+       themes/themes-available/*/*.json \
+       themes/themes-available/*/*.js \
+       themes/themes-available/*/images/logos \
+       || :
+
 # replace /usr/bin/env according to https://fedoraproject.org/wiki/Packaging:Guidelines#Shebang_lines
 sed -e 's%/usr/bin/env perl%/usr/bin/perl%' -i \
     script/thruk_server.pl \
@@ -246,7 +275,7 @@
       /usr/sbin/a2enmod rewrite
     %endif
     %if %{?_unitdir:1}0
-      #%systemd_post thruk.service
+      #systemd_post thruk.service
     %else
       chkconfig --add thruk
     %endif
@@ -509,6 +538,34 @@
 
 
 %changelog
+* Tue Mar 24 2026 Peter Tuschy <foss+rpm@bofh42.de> - 3.20.2-2
+- changed group to 42/addon/naemon for my new repo scripts
+
+* Tue Mar 04 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.20.2-1
+- fix themes build (pin npm package versions, cleanup build artefacts)
+
+* Mon Mar 03 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.20.2-1
+- upstream update
+
+* Mon Mar 03 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- bugfix changelog date and really test source checksum
+
+* Sun Dec 08 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- bugfix post: deactivated systemd_post as intended
+
+* Thu Dec 05 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- removed Requires thruk-selinux (changed Requires order)
+
+* Tue Nov 26 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- reset release number to 1 for my own repo
+- use git source url and save xxh128 hash in the spec file and check it
+- added nodejs as BuildRequires
+- added my new thruk-selinux as Requires for el8+
+
+* Thu Oct 24 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-41
+- added optional macro dist to release
+- Requires perl-interpreter for rhel >= 8 or fedora >= 28 (200 dependencies less)
+
 * Fri Jun 12 2015 Sven Nierlein <sven@consol.de> - 2.00
 - split into several subpackages
 
