--- upstream/specs/thruk.spec	2026-05-12 00:09:39.000000000 +0200
+++ specs/thruk.spec	2026-05-12 17:01:46.467550873 +0200
@@ -8,19 +8,22 @@
 %define apachedir httpd
 %endif
 
+Summary:       Monitoring Webinterface for Nagios/Naemon/Icinga and Shinken
 Name:          thruk
-Version:       3.26
-Release:       46.18
+Version:       3.24
+Release:       1%{?dist}
 License:       GPL-2.0-or-later
-Packager:      Sven Nierlein <sven.nierlein@consol.de>
-Vendor:        Labs Consol
+Group:         42/addon/naemon
+
 URL:           http://thruk.org
-%define fullname %{name}-%{version}
-Source0:       %{fullname}.tar.gz
+Source0:       https://download.thruk.org/pkg/v%{version}/src/%{name}-%{version}.tar.gz
+# this needs to be updated for every version change
+%global src0sum fd3aa66dae0d4458ea4a5b6f32fab0e7
+
 BuildRoot:     %{_tmppath}/%{name}-%{version}-%{release}
-Group:         Applications/Monitoring
-Summary:       Monitoring Webinterface for Nagios/Naemon/Icinga and Shinken
 AutoReqProv:   no
+
+BuildRequires: xxhash
 BuildRequires: make, patch
 BuildRequires: perl
 BuildRequires: perl(Module::Install)
@@ -58,9 +61,14 @@
 
 %package base
 Summary:         Thruk Gui Base Files
-Group:           Applications/System
+Group:           42/addon/naemon
 AutoReqProv:     no
-Requires:        perl logrotate curl
+%if 0%{?rhel} >= 8 || 0%{?fedora} >= 28
+Requires:        perl-interpreter
+%else
+Requires:        perl
+%endif
+Requires:        logrotate  curl
 Requires:        libthruk >= 3.24
 Requires(preun): libthruk
 Requires(post):  libthruk
@@ -74,8 +82,8 @@
 %endif
 %endif
 
-# >=rhel7 and fedora
-%if 0%{?el7}%{?el8}%{?el9}%{?fedora}
+# >=rhel8 and fedora
+%if 0%{?rhel} >= 8 || 0%{?fedora} >= 28
 BuildRequires: httpd
 Requires:      httpd mod_fcgid cronie
 %endif
@@ -86,7 +94,7 @@
 
 %package plugin-reporting
 Summary:     Thruk Gui Reporting Addon
-Group:       Applications/System
+Group:       42/addon/naemon
 Requires:    %{name}-base = %{version}-%{release}
 AutoReqProv: no
 
@@ -95,7 +103,8 @@
 and event reporting.
 
 %prep
-%setup -q -n %{fullname}
+echo "%{src0sum}  %{SOURCE0}" | xxh128sum -c
+%setup -q -n %{name}-%{version}
 
 %build
 export PERL5LIB=/usr/lib/thruk/perl5:/usr/lib64/thruk/perl5
@@ -237,7 +246,7 @@
       /usr/sbin/a2enmod rewrite
     %endif
     %if %{?_unitdir:1}0
-      #%systemd_post thruk.service
+      #systemd_post thruk.service
     %else
       chkconfig --add thruk
     %endif
@@ -475,7 +484,7 @@
 %attr(0755,root,root) %{_datadir}/thruk/script/pnp_export.sh
 %attr(0755,root,root) %{_datadir}/thruk/script/convert_old_datafile
 %attr(0755,root,root) %{_datadir}/thruk/script/check_thruk_rest
-%attr(0755,root,root) %{_datadir}/thruk/script/check_thruk_rest.sh
+#attr(0755,root,root) %{_datadir}/thruk/script/check_thruk_rest.sh
 %{_datadir}/thruk/support
 %{_datadir}/thruk/root
 %{_datadir}/thruk/templates
@@ -515,6 +524,41 @@
 
 
 %changelog
+* Tue May 12 2026 Peter Tuschy <foss+rpm@bofh42.de> - 3.24-1
+- spec now based on upstream 3.26 version
+- upstream update 3.24
+
+* Tue Apr 28 2026 Peter Tuschy <foss+rpm@bofh42.de> - 3.22.2-1
+- upstream update 3.22.2
+
+* Tue Mar 24 2026 Peter Tuschy <foss+rpm@bofh42.de> - 3.20.2-2
+- changed group to 42/addon/naemon for my new repo scripts
+
+* Tue Mar 04 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.20.2-1
+- fix themes build (pin npm package versions, cleanup build artefacts)
+
+* Mon Mar 03 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.20.2-1
+- upstream update
+
+* Mon Mar 03 2025 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- bugfix changelog date and really test source checksum
+
+* Sun Dec 08 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- bugfix post: deactivated systemd_post as intended
+
+* Thu Dec 05 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- removed Requires thruk-selinux (changed Requires order)
+
+* Tue Nov 26 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-1
+- reset release number to 1 for my own repo
+- use git source url and save xxh128 hash in the spec file and check it
+- added nodejs as BuildRequires
+- added my new thruk-selinux as Requires for el8+
+
+* Thu Oct 24 2024 Peter Tuschy <foss+rpm@bofh42.de> - 3.18-41
+- added optional macro dist to release
+- Requires perl-interpreter for rhel >= 8 or fedora >= 28 (200 dependencies less)
+
 * Fri Jun 12 2015 Sven Nierlein <sven@consol.de> - 2.00
 - split into several subpackages
 
