diff -Naur nagflux-0.5.8/config.gcfg.example nagflux-0.5.8.patch/config.gcfg.example
--- nagflux-0.5.8/config.gcfg.example	2026-04-28 11:26:50.000000000 +0200
+++ nagflux-0.5.8.patch/config.gcfg.example	2026-05-20 16:25:48.778970663 +0200
@@ -1,13 +1,7 @@
 [main]
-    # This option is deprecated, use NagiosSpoolfile.Folder when possible
-    NagiosSpoolfileFolder = "/var/spool/nagios"
-    # This option is deprecated, use NagiosSpoolfile.WorkerCount when possible
-    NagiosSpoolfileWorker = 1
     InfluxWorker = 2
     MaxInfluxWorker = 5
     DumpFile = "nagflux.dump"
-    # This option is deprecated, use NagfluxSpoolfile.Folder when possible
-    NagfluxSpoolfileFolder = "/var/spool/nagflux"
     FieldSeparator = "&"
     BufferSize = 10000
     FileBufferSize = 65536
@@ -48,19 +42,19 @@
 [Monitoring]
     # leave empty to disable
     # PrometheusAddress = ":8080"
-    PrometheusAddress = ":8080"
+    PrometheusAddress = ""
 
 [Livestatus]
     Enabled = true
     # tcp or file
-    Type = "tcp"
+    Type = "file"
     # tcp: 127.0.0.1:6557 or file /var/run/live
-    Address = "127.0.0.1:6557"
+    Address = "/var/cache/naemon/live"
     # The amount to minutes to wait for livestatus to come up, if set to 0 the detection is disabled
     MinutesToWait = 2
     # Set the Version of Livestatus. Allowed are Nagios, Icinga2, Naemon.
     # If left empty Nagflux will try to detect it on it's own, which will not always work.
-    Version = ""
+    Version = "Naemon"
 
 [NagiosSpoolfile]
     Enabled = true
@@ -97,8 +91,8 @@
     HostcheckAlias = "hostcheck"
     ClientTimeout  = 5
 
-[InfluxDB "nagflux"]
-    Enabled = true
+[InfluxDB "nagflux1"]
+    Enabled = false
     Version = 1.0
     Address = "http://127.0.0.1:8086"
     Arguments = "precision=ms&u=root&p=root&db=nagflux"
diff -Naur nagflux-0.5.8/nagflux.service nagflux-0.5.8.patch/nagflux.service
--- nagflux-0.5.8/nagflux.service	2026-04-28 11:26:50.000000000 +0200
+++ nagflux-0.5.8.patch/nagflux.service	2026-05-20 16:59:22.882456547 +0200
@@ -6,7 +6,7 @@
 [Service]
 User=root
 Group=root
-ExecStart=/opt/nagflux/nagflux -configPath /opt/nagflux/config.gcfg
+ExecStart=/usr/sbin/nagflux -configPath /etc/nagflux/config.gcfg
 Restart=on-failure
 
 [Install]
